### PR TITLE
Fix: double main tags

### PR DESCRIPTION
**Pull request by math-GH**
Fixed this double `main`
grafik

Changes proposed in this pull request:

- fix the double `<main>` tag in feed views
- improve the `<main>` usage. Now it includes the main content without navigation

How to test the feature manually:

1. normal/global/reader view has unique `<main>`

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/views/auth/formLogin.phtml [hunk 1]
```diff
@@ -39,4 +39,4 @@
 	</form>
 
 	<p><a href="<?= _url('index', 'about') ?>"><?= _t('gen.freshrss.about') ?></a></p>
-</div>
+	</main>
```
Comment (Frenzie) [inline app/views/auth/formLogin.phtml]: Nit.

File: app/views/configure/archiving.phtml [hunk 2]
```diff
@@ -162,4 +162,4 @@
 		</div>
 	</div>
 	<?php endif; ?>
-</div>
+	</main>
```
Comment (Frenzie) [inline app/views/configure/archiving.phtml]: Nit.

File: app/views/subscription/bookmarklet.phtml [hunk 3]
```diff
@@ -15,4 +15,4 @@
 	<h2><?= _t('sub.api.title') ?></h2>
 	<p><?= _t('sub.api.documentation') ?></p>
 	<kbd><?= Minz_Url::display(array('c' => 'feed', 'a' => 'add'), 'html', true) ?>&amp;url_rss=%s</kbd>
-</div>
\ No newline at end of file
+</main>
```
Comment (Frenzie) [inline app/views/subscription/bookmarklet.phtml]: I think we prefer newline at end of file, or is that just me? In any case I prefer not seeing git/diff talking about it. ;-) (Lesser nit.)

File: app/views/update/checkInstall.phtml [hunk 4]
```diff
@@ -38,4 +38,4 @@
 	<?php } ?>
 	*/ ?>
 
-</div>
+	</main>
```
Comment (Frenzie) [inline app/views/update/checkInstall.phtml]: Nit.

File: app/views/update/index.phtml [hunk 5]
```diff
@@ -47,4 +47,4 @@
 	<?php if ($this->update_to_apply) { ?>
 	<a class="btn btn-important" href="<?= _url('update', 'apply') ?>"><?= _t('admin.update.apply') ?></a>
 	<?php } ?>
-</div>
+	</main>
```
Comment (Frenzie) [inline app/views/update/index.phtml]: Nit.

**Review by Frenzie — COMMENTED** [review]
Should be good, left a couple of minor formatting nits.

File: app/views/subscription/bookmarklet.phtml (hunk 3 above)
Comment (Alkarex) [reply app/views/subscription/bookmarklet.phtml]: New line at the end of the file indeed, and that should be automatic in editors supporting https://github.com/FreshRSS/FreshRSS/blob/edge/.editorconfig

**Comment by Alkarex** [comment]
Prioritised for 1.19.0 because it seems to be a regression from https://github.com/FreshRSS/FreshRSS/pull/3747, right?

File: app/views/subscription/bookmarklet.phtml (hunk 3 above)
Comment (math-GH) [reply app/views/subscription/bookmarklet.phtml]: added the newline.
(it was not before there)

**Comment by math-GH** [comment]
> Prioritised for 1.19.0 because it seems to be a regression from #3747, right?

yes

**Pull request merged**